### PR TITLE
Front-end snags list #1

### PR DESCRIPTION
**Pull request by froddd**
- Changes slug used to trigger regions map on Investment landing page to `with-regions-map` as per the CMS help text.
- Ensures map takes precedence over any other content, in case it has been wrongly set, rather than displaying it alongside.
- Ensure card images span whole width
- Only show Region Spotlight if there is only one related region
- Show sector and sub-sectors if set

To do (delete all that do not apply):

 - [ ] Change has a jira ticket that has the correct status.
 - [ ] Changelog entry added.
 - [ ] (if there are vulnerable requirements) Upgraded any vulnerable dependencies.
 - [ ] (if updating requirements) Requirements have been compiled.
 - [x] (if adding CSS) CSS have been compiled.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [ ] (if changing content) Locale files have been updated and translated strings have been added if available.
 - [ ] (if adding env vars) Added any new environment variable to vault.
 - [ ] (if adding feature flags) Cleaned up old flags
 - [ ] (if changing the UI) Add a printscreen to the PR

### CODE REVIEW

File: core/tests/test_views.py [hunk 1]
```diff
@@ -1392,6 +1392,29 @@ def test_getting_keyed_region_data_on_region_listing_page(
                'full_path'] == '/international/content/about-uk/regions/northern-ireland/'
 
 
+@patch('directory_cms_client.client.cms_api_client.lookup_by_path')
+def test_atlas_opportunity_page_sectors_label(mock_cms_response, rf):
```
Comment (stevejalim) [inline core/tests/test_views.py:1396]: Nice one :)

File: investment_atlas/templates/investment_atlas/opportunity.html [hunk 2]
```diff
@@ -4,6 +4,7 @@
 {% block css_layout_class %}atlas atlas-regular{% endblock css_layout_class %}
 
 {% block content %}
+    {% include 'investment_atlas/includes/debug_data.html' with data=page %}
```
Comment (stevejalim) [inline investment_atlas/templates/investment_atlas/opportunity.html:7]: One to come out?

```suggestion
```

**Review by stevejalim — CHANGES_REQUESTED** [review]
Overall, Python/Django stuff looks fab, thanks!

Changes requested because there's a debug line left in there + I think it needs rebasing on develop and assets recompiling

**Review by bobmeredith — COMMENTED** [review]
Looks good to me

**Review by stevejalim — APPROVED** [review]

**Pull request merged**